apps/sms/services/js/moz_settings/moz_settings_client.js

```diff
@@ -0,0 +1,83 @@
+/* global bridge,
+          BroadcastChannel */
+
+(function(exports) {
+'use strict';
+
+/**
+ * Name of the service that is responsible for settings functionality.
+ * @type {string}
```

Comment:
__optional nit (:bike:):__ you can also add ```@const``` directive to the jsdoc for the constants, but it's optional - this file is marked fully green by my IDE - it's a rare thing :D